### PR TITLE
Simplify config sub-section menu items; See: websharks/comment-mail#187

### PR DESCRIPTION
**Pull request by kristineds**
Simplify config sub-section menu items; 

- Rename "Restore Default Options" to "Restore", move designated icon to end of the text, and convert text link to blue button (green on hover)
- Add "Beta Testers" text link with icon
- Move "Restore" button, "Newsletter","Beta Testers", "Pro Upgrade", "Preview Pro Features" text link to upper right side of plugin options panel

![Simplify sub section menu items - v1.0 (Lite)]
(https://cloud.githubusercontent.com/assets/7514953/12355317/c1dedf4c-bbd6-11e5-8412-686d98c0b27e.png)

See: websharks/comment-mail#187

### CODE REVIEW

**Comment by kristineds** [comment]
_NOTE: This is a work-in-progress. Still working on the following:_

- Display installed version number on plugin options panel

**Comment by kristineds** [comment]
@raamdev @jaswsinc This is ready for review
- Show installed version number (with link to changelog): https://github.com/websharks/comment-mail/commit/3bf3535d1f25c87172c11fb567d6e8312956e9a5

screen shot 2016-01-27 at 12 47 40 am

**Comment by raamdev** [comment]
@kristineds That looks awesome! Can we get a PR for the Pro version as well? :-)

File: comment-mail/includes/classes/menu-page.php [hunk 1]
```diff
@@ -4361,6 +4356,24 @@ protected function heading($title, $logo_icon = '')
 
 					$heading .= '  <a href="'.esc_attr($this->plugin->utils_url->product_page()).'" target="_blank"><i class="fa fa-heart-o"></i> '.__('Pro Upgrade', 'comment-mail').'</a>'."\n";
 				}
+
+				$heading .= '     <a href="'.esc_attr($this->plugin->utils_url->subscribe_page()).'" target="_blank"><i class="fa fa-envelope-o"></i> '.__('Newsletter (Subscribe)', 'comment-mail').'</a>'."\n";
+				$heading .= '     <a href="'.esc_attr($this->plugin->utils_url->tester_page()).'" target="_blank"><i class="fa fa-envelope-o"></i> '.__('Beta Testers', 'comment-mail').'</a>'."\n";
+				$heading .= '  </div>'."\n";
+
+				$heading .= '  <div class="pmp-version">'."\n";
+				$heading .= '     <span> '.sprintf(__('%1$s&trade; Pro v%2$s (<a href="https://comment-mail.com/changelog/">changelog</a>)', 'comment-mail'), esc_html($this->plugin->name), esc_html($this->plugin->version)).'</span>'."\n";
```
Comment (raamdev) [inline comment-mail/includes/classes/menu-page.php]: @kristineds This shouldn't say "Pro" for the Lite version...

**Comment by kristineds** [comment]
@raamdev "Pro" text removed on version number. https://github.com/websharks/comment-mail/commit/4ffbc2fa9fb7920e56d3386692745ddaf1b6c8c9

screen shot 2016-01-27 at 1 36 00 am

**Comment by raamdev** [comment]
Changing this to `in progress` until there's some more clarity around the status of this PR and its Pro counterpart; see https://github.com/websharks/comment-mail-pro/pull/46#issuecomment-178092301.

**Comment by kristineds** [comment]
@raamdev I posted my PR for the Pro version [here](https://github.com/websharks/comment-mail-pro/pull/46#issuecomment-179668273)

**Comment by raamdev** [comment]
@kristineds Thank you! I reviewed this and the Pro PR; here's some feedback:

The group of links with `text-align: center;` doesn't look right; there's too much space to the right of the links:



I'd like all of those links and the Restore button to line up with on the right side. I also suggest increasing the `line-height` to `1.75em` for the links to separate them out a bit. This is how it should look when finished:



----

I'm making a similar request for the Pro version; I'll post screenshots on that PR.

**Comment by kristineds** [comment]
@raamdev Submitting PR for Lite version: https://github.com/websharks/comment-mail/commit/bf3c64960ad39af453a2963a448d8c7011b4fef6 & https://github.com/websharks/comment-mail/commit/4ac4b5ad1e44b0bc5608f23c8fd654d6ccc33ec5 
_(Forgot about the Restore button so I have to push another commit for this)_

screen shot 2016-02-07 at 12 36 41 am

**Comment by raamdev** [comment]
@kristineds I tested this PR and the alignment looks great now! Thank you.

**Pull request closed without merging**